openfisca_aotearoa/tests/rates_rebates.yaml

```diff
@@ -71,4 +71,12 @@
   output_variables:
     maximum_income_for_full_rebate: 41742
     minimum_income_for_no_rebate: 46702
-
+- name: Rounding issue for Tauranga 2018 launch 
```

Comment:
Should be more descriptive and less coincidental: reading this test name makes me feel like I should remove it from the codebase one year from now. Conversely, I probably would not expect it to fail when I remove the non-legal `floor` and not know if I've made a mistake.

What about something like “Replicate rounding behaviour from agencies' existing infrastructure”?